sos/plugins/kubernetes.py

```diff
@@ -22,33 +23,101 @@ class kubernetes(Plugin, RedHatPlugin):
     """Kubernetes plugin
     """
 
-    option_list = [("podslog", "capture logs for pods", 'slow', False)]
+    # Red Hat Atomic Platform and OpenShift Enterprise use the
+    # atomic-openshift-master package to provide kubernetes
+    packages = ('kubernetes', 'atomic-openshift-master')
```

Comment:
Containerized installs will not have these packages, so we probably want to also add a `files` tuple pointing at configuration files to trigger auto-enable in systems that have a kubernetes master running inside a container